MHIdentityKitTests/Infrastructure/OAuth2IdentityManagerTests.swift

```diff
@@ -11,73 +11,74 @@ import Foundation
 import XCTest
 @testable import MHIdentityKit
 
+@available(iOS 13, tvOS 13.0.0, macOS 10.15, *)
```

Comment:
This test should not be changed.

Reply:
So to bring it back as it was before?

Reply:
Yes, i have no idea why you changed that test.